src/Orchard/Utility/Extensions/HttpRequestExtensions.cs

```diff
@@ -85,6 +85,10 @@ public static class HttpRequestExtensions {
                 return true;
             }
 
+            if (url.StartsWith("../")) {

```

Comment:
This is just something to get some test code working. It may go to a different PR.